src/jnv.rs

```diff
@@ -74,16 +74,16 @@ impl Jnv {
                 history: Default::default(),
                 prefix: String::from("❯❯ "),
                 mask: Default::default(),
-                prefix_style: StyleBuilder::new().fgc(Color::DarkCyan).build(),
-                active_char_style: StyleBuilder::new().bgc(Color::DarkMagenta).build(),
+                prefix_style: StyleBuilder::new().fgc(Color::Cyan).build(),
```

Comment:
@corneliusroemer I would like to proceed with the merge, but please make a revision just regarding this https://github.com/ynqa/jnv/issues/3#issuecomment-2008950603.

Reply:
Thanks for your feedback! I'm not quite sure what exactly I should change here, can you make that change? You should have push rights to the branch!

Reply:
I've just made a change, turning one of the Cyans into Blue so that each color is used only once again